[20106] Support configuring data representation version (#73)
eProsima/Shapes-Demo-Docs: docs/first_steps/first_steps.rst

  The latter will send that last stored values to subscribers joining after the publisher has been created. (see
  `Fast DDS DurabilityQosPolicy Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/standardQosPolicies.html#durabilityqospolicy>`_).

> - **Data representation:** The Data representation version is set to version 1,

Reviewer: ```suggestion
- **Data Representation:** The Data Representation QoS policy is set to "XCDR" (version 1),
```

> - **Data Representation:** The Data Representation Qos Policy is set to ``XCDR``

Reviewer: ```suggestion
- **Data Representation:** The Data Representation QoS Policy is set to ``XCDR``
```

>   ``XCDR``, but the user can select the representation version 2, ``XCDR2``.

Reviewer: ```suggestion
  but the user can optionally select the ``XCDR2`` Data Representation.
```

  (see

>   `Fast DDS Data representation Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/api_reference/dds_pim/core/policy/datarepresentationid.html>`_).

Reviewer: It would be better redirecting to the explanatory section instead of the `api-reference` 
https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/xtypesExtensions.html#datarepresentationqospolicy
